samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -2334,6 +2334,34 @@
           }
         ]
       }
+    },
+    "Route53ChangeResourceRecordSetsPolicy": {
+      "Description": "Gives permission to change resource records sets in Route53",
+      "Parameters": {
+        "HostedZoneId": {
+          "Description": "ID of the hosted zone"
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "route53:ChangeResourceRecordSets"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:route53:${AWS::Region}:${AWS::AccountId}:hostedzone/${HostedZoneId}",
```

Comment:
This resource arn does not seem to require Region and AccountId.
See: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/r53-api-permissions-ref.html#required-permissions-resource-record-sets

Reply:
Sorry, thought it also need Region and AccountId.

I'll change it to "arn:${AWS::Partition}:route53:::hostedzone/${HostedZoneId}", and updated the code/tests.